CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+When contributing to this repository, please keep in mind the following information.
+
+We highly encourage you to join our [Slack channel on the UIUC OSS Slack](https://illinois-oss.slack.com) to discuss features, seek development support, and be alerted about our bi-weekly dev meetups. 
+
+## To develop on your machine
+
+1. Make a fork of the repository by clickin the fork button at the top right of the page
```

Comment:
```suggestion
1. Make a fork of the repository by _clickin_ the fork button at the top right of the page
```
Me likes 🤪

Reply:
Typo :(

Reply:
Let's keep it! <3

Reply:
I just fixed the typo....but I'll change it back.